charts/sequencer/templates/configmaps.yaml

```diff
@@ -60,6 +60,7 @@ data:
   ASTRIA_SEQUENCER_ENABLE_MINT: "false"
   # Socket address for gRPC server
   ASTRIA_SEQUENCER_GRPC_ADDR: "0.0.0.0:{{ .Values.ports.sequencerGRPC }}"
+  ASTRIA_SEQUENCER_COMETBFT_RPC_ADDR: "http://127.0.0.1:{{ .Values.ports.cometBFTRPC }}"
```

Comment:
nit: add to the `else` section of `if not .Values.global.dev` below instead of here, makes it clear we have breaking changes when doing a new main release.